pkg/spec/cluster.go

```diff
@@ -188,6 +195,11 @@ func (c *ClusterSpec) Cleanup() {
 	if len(c.Version) == 0 {
 		c.Version = defaultVersion
 	}
+
+	if len(c.BaseImage) == 0 {
```

Comment:
Can you put this change before `if len(c.Version) == 0` or after `c.Version =` ?